In-house patch
makes the original "00-foomatic-kitload.in.patch" more general
...to be promote-able to community.

--- a/configure.ac	2016-07-20 00:40:31.000000000 -0700
+++ b/configure.ac	2016-07-20 14:02:30.873396304 -0700
@@ -14,6 +14,7 @@
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
+AC_CHECK_PROG(TAR,tar,tar)
 
 dnl # Checks for libraries.
 dnl AC_PATH_XTRA
